config/typescript/modules.d.ts

```diff
@@ -5,3 +5,27 @@ declare module 'url' {
 
   export function parse(url: string): URLDetails;
 }
+
+declare module 'enzyme-adapter-react-16' {
```

Comment:
Why is this needed?

Reply:
what's wrong with the community one? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/enzyme-adapter-react-16/index.d.ts